src/scripts/mkarv

```diff
@@ -570,6 +570,9 @@ def write_metrics(indent, metrics):
 if __name__ == '__main__':
     args = parse_arguments()
 
+    if args.template_directory == 'auto':
```

Comment:
Looking at this now, it maybe nicer to use `None`, but that would look confusing in the `--help` output.